cms/envs/production.py

```diff
@@ -375,6 +375,18 @@ def get_env_setting(setting):
 
 DATABASES = AUTH_TOKENS['DATABASES']
 
+# If DB_CONFIG_OVERRIDE_DEFAULT is set, use the named config instead of default
+# Useful for overriding the config for the celery workers
+DB_CONFIG_OVERRIDE_DEFAULT = os.environ.get('DB_CONFIG_OVERRIDE_DEFAULT', None)
```

Comment:
Using env vars to set the app configuration breaks a lot of assumptions about what is running in production based on remote config settings